A lot of updates (#1)
ah01/arduino-stuff: readme.md

# *Arduino* Stuff #

> In tis repo you can file multiple arduino rpojects which are to small for stan-alone repos.

Reviewer: ```suggestion
In this repo you can find multiple Arduino projects which are too small for standalone repos.
```

But that doesnt mean that they are not interesting!

## Smart rgb strip

> In this project you will learn how to control your smart rgb led strip with arduino YUN.

Reviewer: ```suggestion
In this project you will learn how to control your smart RGB LED strip with Arduino Yun.
```

[Picture](https://twitter.com/horcicaa/status/381399934102036480) 

## Smart Led blinking

> This folder contains code how to make smart led. Using two buttons and led you can make flashlight that learns how to blink! (Spoiler: Thats not AI...)

Reviewer: ```suggestion
This folder contains code to make a smart LED. Using two buttons and an LED you can make a flashlight that learns how to blink! (Spoiler: That's not AI...)
```
